js/views/me.vue

```diff
@@ -81,6 +85,9 @@ export default  {
         this._handler.remove();
     },
     methods: {
+        change_password() {
+            document.location = '/change';
```

Comment:
This won't work if udata is served from a subfolder, does it?

Reply:
Yes but right now, it already can't be served in a subfolder because this pattern is already used in many places.

Reply:
Ok :-)